### ISSUE TITLE
dom-if contents show regardless of whether condition is true or false

### ISSUE THREAD

**Issue by hguiney** [Polymer/polymer#4643]
#### Live Demo

http://jsbin.com/quhezacofo/edit?html,output

#### Steps to Reproduce
1. Use two `dom-if`s in a template.
2. Put something different in each `dom-if`.
3. Point one `dom-if`’s `if` attribute to an expression that evaluates to `true`; the other `false`.

#### Expected Results
Conditional loading of the different things.

#### Actual Results
Both things load every time.

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [ ] <del>Edge</del> (not tested)
- [x] Safari 10
- [ ] <del>IE 11</del> (not tested)

### Versions

- Polymer: v2.0.0
- webcomponents: v1.0.1

**Comment by Westbrook** [comment]
Looks like if you pull out some of the spaces in your binding this issue does away: http://jsbin.com/qedapijulo/edit?html,output

tldr; `{{isMultipleChoice(question.type)}}` instead of `{{isMultipleChoice( question.type )}}`...

**Comment by hguiney** [comment]
Coming from Angular development I don’t expect this kind of behavior at all. I would assume that whitespace around parameters for readability is irrelevant, as it is in regular JavaScript. (On that note, I also expect to be able to do comparison directly in the `if` attribute rather than outsourcing it to a helper method, but I digress.) I see this is P1 now, but if it does turn out to be expected behavior I think it would be crucial to see that reflected in the docs. Thank you.

**Comment by kevinpschaaf** [comment]
There is a bug in the parsing, will be fixed.
